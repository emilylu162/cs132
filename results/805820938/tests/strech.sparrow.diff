1,2c1,4
< The Sparrow-V program is invalid
< Error: The program has no function declarations
---
> 100
> 40464
> 80770
> 121302
