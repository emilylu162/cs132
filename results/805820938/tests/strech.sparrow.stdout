func Main()
t0 = @f
t2 = t0
t0 = t2
t0 = call t0()
t3 = t0
t0 = t3
print(t0)
v1_1 = t3
      return v1_1

func f()
s3_stack = s3
s4_stack = s4
s11_stack = s11
s5_stack = s5
s10_stack = s10
s6_stack = s6
s7_stack = s7
s8_stack = s8
s9_stack = s9
s1_stack = s1
s2_stack = s2
t0 = 10
s1 = t0
t0 = 20
t4 = t0
t0 = s1
t1 = t4
t0 = t0 + t1
t5 = t0
t0 = 30
s2 = t0
t0 = s1
t1 = s2
t0 = t0 < t1
s3 = t0
t0 = s3
if0 t0 goto L8
t0 = s1
t1 = t5
t0 = t0 + t1
t2 = t0
t0 = s1
t1 = t4
t0 = t0 + t1
t3 = t0
goto L10
L8:
t0 = t4
t1 = t5
t0 = t0 + t1
s4 = t0
goto L14
L10:
t0 = 50
s5 = t0
t0 = t2
t1 = s5
t0 = t0 + t1
t4 = t0
t0 = t5
t1 = t2
t0 = t0 + t1
s6 = t0
t0 = s6
t1 = t3
t0 = t0 + t1
s7 = t0
t0 = s7
print(t0)
t0 = t5
t1 = t3
t0 = t0 + t1
s4 = t0
t0 = 40
s8 = t0
t0 = t4
t1 = s8
t0 = t0 + t1
t4 = t0
goto L8
L14:
t0 = 10
s9 = t0
t0 = s1
t1 = s9
t0 = t0 + t1
t3 = t0
L22:
t0 = 55
t4 = t0
t0 = 20
s10 = t0
t0 = s1
t1 = s10
t0 = t0 - t1
t5 = t0
t0 = t4
t1 = t5
t0 = t0 < t1
s11 = t0
t0 = s11
if0 t0 goto L20
t0 = t5
t1 = t4
t0 = t0 + t1
t2 = t0
t0 = t2
t1 = s9
t0 = t0 - t1
s4 = t0
t0 = t2
t1 = s4
t0 = t0 + t1
s3 = t0
t0 = s3
print(t0)
t0 = t5
t1 = s4
t0 = t0 + t1
t4 = t0
t0 = 2
s5 = t0
t0 = s5
t1 = t4
t0 = t0 - t1
t3 = t0
goto L22
L20:
t0 = 7
s7 = t0
t0 = t3
t1 = s7
t0 = t0 + t1
t4 = t0
t0 = t5
t1 = t4
t0 = t0 + t1
t5 = t0
t0 = 2
s5 = t0
t0 = s5
t1 = t5
t0 = t0 * t1
s4 = t0
t0 = t3
t1 = s9
t0 = t0 + t1
s8 = t0
t0 = 55
s1 = t0
t0 = s1
s11 = t0
t0 = @func_g
s3 = t0
t0 = s8
t1 = s11
t0 = t0 < t1
s2 = t0
t0 = s2
if0 t0 goto L27
t0 = s8
t1 = s10
t0 = t0 - t1
s7 = t0
a2_stack = a2
a3_stack = a3
a4_stack = a4
a5_stack = a5
a6_stack = a6
a7_stack = a7
t0 = t4
a2 = t0
t0 = t5
a3 = t0
t0 = t2
a4 = t0
t0 = s4
a5 = t0
t0 = t3
a6 = t0
t0 = s8
a7 = t0
h_2_stack = s11
i_2_stack = s7
t0 = s3
t5_stack = t5
t2_stack = t2
t0 = call t0(h_2_stack i_2_stack)
a2 = a2_stack
a3 = a3_stack
a4 = a4_stack
a5 = a5_stack
a6 = a6_stack
a7 = a7_stack
t5 = t5_stack
t2 = t2_stack
s11 = t0
t0 = s7
t1 = s11
t0 = t0 + t1
s9 = t0
goto L29
L27:
t0 = 1
s1 = t0
t0 = s8
t1 = s1
t0 = t0 + t1
s2 = t0
t0 = s5
t1 = s2
t0 = t0 * t1
s9 = t0
L29:
t0 = 5
s4 = t0
t0 = s9
t1 = s4
t0 = t0 - t1
s3 = t0
t0 = s8
t1 = s3
t0 = t0 + t1
s2 = t0
t0 = s9
t1 = s2
t0 = t0 - t1
s11 = t0
t0 = s11
t1 = s3
t0 = t0 + t1
s1 = t0
t0 = s1
print(t0)
t0 = s9
t1 = s2
t0 = t0 + t1
s7 = t0
t0 = t5
t1 = t2
t0 = t0 + t1
t5 = t0
t0 = t2
t1 = s10
t0 = t0 + t1
s5 = t0
t0 = s7
t1 = s5
t0 = t0 + t1
s4 = t0
t0 = t2
t1 = s4
t0 = t0 < t1
s6 = t0
L3:
t0 = s6
if0 t0 goto L12
t0 = t2
t1 = s5
t0 = t0 + t1
s3 = t0
t0 = 30
s11 = t0
t0 = s3
t1 = s11
t0 = t0 + t1
s1 = t0
t0 = s7
t1 = s1
t0 = t0 - t1
s1 = t0
t0 = s3
t1 = s1
t0 = t0 - t1
s4 = t0
t0 = 0
s9 = t0
t0 = s4
t1 = s9
t0 = t0 * t1
s6 = t0
t0 = t5
t1 = s8
t0 = t0 + t1
s5 = t0
t0 = s3
t1 = s4
t0 = t0 + t1
s4 = t0
t0 = s5
t1 = s4
t0 = t0 - t1
s10 = t0
t0 = s10
print(t0)
goto L3
L12:
t0 = s2
t1 = s4
t0 = t0 - t1
s8 = t0
t0 = s3
t1 = s8
t0 = t0 + t1
s8 = t0
g_2 = s8
s3 = s3_stack
s4 = s4_stack
s11 = s11_stack
s5 = s5_stack
s10 = s10_stack
s6 = s6_stack
s7 = s7_stack
s8 = s8_stack
s9 = s9_stack
s1 = s1_stack
s2 = s2_stack
      return g_2

func func_g(x7_3 x8_3)
s4_stack = s4
s11_stack = s11
s5_stack = s5
s6_stack = s6
s10_stack = s10
s7_stack = s7
s9_stack = s9
s1_stack = s1
s2_stack = s2
t0 = a2
t1 = a3
t0 = t0 + t1
t4 = t0
t0 = a4
t1 = a5
t0 = t0 + t1
t3 = t0
t0 = a6
t1 = a7
t0 = t0 + t1
t5 = t0
t0 = x7_3
t1 = x8_3
t0 = t0 + t1
t2 = t0
t0 = 10
s6 = t0
t0 = s6
s7 = t0
t0 = x8_3
t1 = a2
t0 = t0 < t1
s5 = t0
t0 = s5
if0 t0 goto L42
t0 = @func_g
s11 = t0
a2_stack = a2
a3_stack = a3
a4_stack = a4
a5_stack = a5
a6_stack = a6
a7_stack = a7
t0 = s5
a2 = t0
t0 = x7_3
a3 = t0
t0 = s5
a4 = t0
t0 = a6_stack
a5 = t0
t0 = s6
a6 = t0
t0 = s5
a7 = t0
t0 = a3_stack
x2_3_stack = t0
ten_3_stack = s6
t0 = s11
t4_stack = t4
t3_stack = t3
t5_stack = t5
t2_stack = t2
t0 = call t0(x2_3_stack ten_3_stack)
a2 = a2_stack
a3 = a3_stack
a4 = a4_stack
a5 = a5_stack
a6 = a6_stack
a7 = a7_stack
t4 = t4_stack
t3 = t3_stack
t5 = t5_stack
t2 = t2_stack
t4 = t0
L42:
t0 = t2
t1 = t4
t0 = t0 - t1
s7 = t0
t0 = t3
t1 = s7
t0 = t0 * t1
s1 = t0
t0 = t5
t1 = s1
t0 = t0 - t1
s9 = t0
t0 = t2
t1 = s7
t0 = t0 + t1
s10 = t0
t0 = s7
t1 = s1
t0 = t0 + t1
s2 = t0
t0 = s9
t1 = s10
t0 = t0 + t1
s4 = t0
j_3 = s4
s4 = s4_stack
s11 = s11_stack
s5 = s5_stack
s6 = s6_stack
s10 = s10_stack
s7 = s7_stack
s9 = s9_stack
s1 = s1_stack
s2 = s2_stack
      return j_3


